src/ActorWorld.ts

```diff
@@ -2,15 +2,29 @@ import { World, IWorldOptions } from '@cucumber/cucumber'
 import Actor from './Actor'
 import ActorLookup from './ActorLookup'
 import assignTasks from './assignTasks'
+import asyncAssignTasks from './asyncAssignTasks'
+
+export interface IActorWorldOptions extends IWorldOptions {
+  packageType: 'commonjs' | 'module'
+}
 
 export default class ActorWorld extends World {
   public readonly actorLookup = new ActorLookup()
+  public readonly promise
+
+  constructor(props: IActorWorldOptions) {
+    console.log('>>>> ActorWorld', props)
```

Comment:
Is this debug logging? Could it be removed?

Reply:
yikes! Sorry about that! Yes, it should be removed!